cloudformation/calculation-athena.yaml

```diff
@@ -52,11 +52,51 @@ Resources:
             Parameters:
               {
                 "ignore.malformed.json": true,
-                "serialiazation.format": 1,
+                "serialization.format": 1,
                 "field.delim": "",
               }
             SerializationLibrary: "org.openx.data.jsonserde.JsonSerDe"
 
+  RateTableTable:
+    Type: AWS::Glue::Table
+    Properties:
+      CatalogId: !Ref AWS::AccountId
+      DatabaseName: !Ref CalculationsDB
+      TableInput:
+        Name: btm_rate_tables
+        TableType: EXTERNAL_TABLE
+        Parameters:
+          has_encrypted_data: "true"
+          projection.enabled: "true"
+          projection.datetime.type: date
+          projection.datetime.range: "2022-01-01,NOW"
+          projection.datetime.format: "yyyy-MM-dd"
+          projection.datetime.interval: 1
+          projection.datetime.interval.unit: DAYS
+          storage.location.template:
+            !Join [
+              "",
+              ["s3://", !Ref ConfigBucket, "/rate_tables/prices.csv"],
```

Comment:
It seemed to make more sense.  Happy to discuss.  Should maybe be made consistent with the file name as well.

Reply:
I'm really not fussed about what to call them; for now I'm inclined to leave it as "rate tables", just because that's how we refer to it in a lot of other places outside of the code, including (I believe) in discussions b/w our PMs and GDS.  I'll bring it up to them and it will be simple enough to put a change in later if necessary I think.

Interestingly, I went looking for a definition of 'rate table', and it looks like it comes [from the USA](https://en.wikipedia.org/wiki/Rate_schedule_(federal_income_tax)) (the IRS, in fact), and seems to mean something like what we're doing.  So that might explain why it doesn't seem unusual to me...